Broken manual GH release page action

**What happened:**
The _manual gh release page_ action is broken. 
Example : https://github.com/gardenlinux/gardenlinux/actions/runs/17494152244/job/49690909546

```
2025-09-05T13:15:07.0728917Z ##[group]Run .github/workflows/release_note.py create --tag 1592.13 --commit "$(echo "2b302ec2abc9275921e1321074b9d6f566860fd1")"
2025-09-05T13:15:07.0729957Z [36;1m.github/workflows/release_note.py create --tag 1592.13 --commit "$(echo "2b302ec2abc9275921e1321074b9d6f566860fd1")"[0m
2025-09-05T13:15:07.0760286Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2025-09-05T13:15:07.0760652Z env:
2025-09-05T13:15:07.0760922Z   pythonLocation: /opt/hostedtoolcache/Python/3.13.7/x64
2025-09-05T13:15:07.0761383Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.13.7/x64/lib/pkgconfig
2025-09-05T13:15:07.0761839Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.7/x64
2025-09-05T13:15:07.0762249Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.7/x64
2025-09-05T13:15:07.0762658Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.7/x64
2025-09-05T13:15:07.0763065Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.13.7/x64/lib
2025-09-05T13:15:07.0763687Z   GITHUB_TOKEN: ***
2025-09-05T13:15:07.0763925Z ##[endgroup]
2025-09-05T13:15:07.0818519Z /home/runner/work/_temp/d3b41e44-4958-44a9-beec-d3ed6ff86047.sh: line 1: .github/workflows/release_note.py: No such file or directory
2025-09-05T13:15:07.0824719Z ##[error]Process completed with exit code 127.
```
 

The problem was introduced via https://github.com/gardenlinux/gardenlinux/commit/c34c44295837d35afc7b0ec2f71ebf970ef13255, mainly the removal of the custom path for the _flavors.yaml_ checkout step.

--- a/.github/workflows/manual_gh_release_page.yml
+++ b/.github/workflows/manual_gh_release_page.yml
@@ -55,8 +55,7 @@ jobs:
         run: |
           echo "Give GLVD some time to ingest the data. It is not clear how long this takes exactly, and there is no single indicator for when it was done."
           sleep 180
-
-  github_release:
+  flavors_matrix:
     needs: glvd
     runs-on: ubuntu-24.04
     defaults:
@@ -64,16 +63,7 @@ jobs:
         shell: bash
     outputs:
       matrix: ${{ steps.matrix.outputs.matrix }}
-    permissions:
-      id-token: write
-      contents: write
-    environment: oidc_aws_s3_upload
     steps:
-      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # pin@v5.0.0
-        with:
-          submodules: true
-      - name: install dependencies for generate_release_note.py script
-        run: sudo apt-get update && sudo apt-get install -qy --no-install-recommends python3-boto3
       - name: Checkout flavors of the release commit
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # pin@v5.0.0
         with:
@@ -88,6 +78,23 @@ jobs:
         uses: gardenlinux/python-gardenlinux-lib/.github/actions/flavors_parse@033901ecf380b9241117778ca6c369567a09494b # pin@0.8.9
         with:
           flags: '--no-arch --json-by-arch --publish'
+  github_release:
+    needs: glvd
+    runs-on: ubuntu-24.04
+    defaults:
+      run:
+        shell: bash
+    permissions:
+      id-token: write
+      contents: write
+    environment: oidc_aws_s3_upload
+    steps:
+      - name: Checkout full repository (current branch)
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # pin@v5.0.0
+        with:
+          submodules: true
+      - name: install dependencies for generate_release_note.py script
+        run: sudo apt-get update && sudo apt-get install -qy --no-install-recommends python3-boto3
       - name: create GitHub release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
@@ -99,7 +106,7 @@ jobs:
           path: .github_release_id
 
   github_release_artifacts_upload:
-    needs: github_release
+    needs: [github_release, flavors_matrix]
     runs-on: ubuntu-24.04
     permissions:
       contents: write
@@ -109,7 +116,7 @@ jobs:
       CNAME: ''
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.github_release.outputs.matrix ) }}
+      matrix: ${{ fromJson(needs.flavors_matrix.outputs.matrix ) }}
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # pin@v5.0.0
         with:

